model/preference.go

```diff
@@ -33,7 +33,9 @@ const (
 
 	PREFERENCE_CATEGORY_NOTIFICATIONS = "notifications"
 	PREFERENCE_NAME_EMAIL_INTERVAL    = "email_interval"
-	PREFERENCE_DEFAULT_EMAIL_INTERVAL = "30" // default to match the interval of the "immediate" setting (ie 30 seconds)
+
+	PREFERENCE_EMAIL_INTERVAL_IMMEDIATE       = "30"  // the "immediate" setting is actually 30s
+	PREFERENCE_EMAIL_INTERVAL_FIFTEEN_MINUTES = "900" // fifteen minutes is 900 seconds
```

Comment:
Maybe don't include "FIFTEEN_MINUTES" in the constant name. If we need to change it for whatever reason (to say 20 minutes), we don't want to have to update the constant name everywhere. I know the client does it but I don't think it's a good idea